spanner/api/Spanner.Samples.Tests/QueryDataWithNumericParameterTest.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class QueryDataWithNumericParameterTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public QueryDataWithNumericParameterTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestQueryDataWithNumericParameter()
+    {
+        await _spannerFixture.CreateVenuesTableAndInsertDataAsync();
+
+        AddNumericColumnAsyncSample addColumnSample = new AddNumericColumnAsyncSample();
+        await addColumnSample.AddNumericColumnAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.DatabaseId);
+
+        UpdateDataWithNumericAsyncSample updateNumericSample = new UpdateDataWithNumericAsyncSample();
+        await updateNumericSample.UpdateDataWithNumericAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.DatabaseId);
```

Comment:
Let me know if it makes more sense to add these to the fixture. I didn't because I didn't want to slow down all other tests since this setup is only required for this test.

Reply:
This is fine here, if in the future it is need for more samples, we can move it to the fixture then.